web/geo/update-world.js

```diff
@@ -1,20 +1,46 @@
 const path = require('path');
 
 const { validateGeometry } = require('./validate');
+const { validateGeometryV2 } = require('./validate-v2');
 const { getJSON } = require('./utilities');
 const { generateTopojson } = require('./generate-topojson');
+const { generateAggregates } = require('./generate-aggregates');
+const { generateExchangesToIgnore } = require('./generate-exchanges-to-exclude');
+const { getZonesJson } = require('./files');
 
 const config = {
   WORLD_PATH: path.resolve(__dirname, './world.geojson'),
   OUT_PATH: path.resolve(__dirname, '../src/world.json'),
   ERROR_PATH: path.resolve(__dirname, '.'),
-  MIN_AREA_HOLES: 600000000,
+  MIN_AREA_HOLES: 5000000,
   MAX_CONVEX_DEVIATION: 0.708,
   MIN_AREA_INTERSECTION: 500000,
-  SLIVER_RATIO: 0.01, // ratio of length and area to determine if the polygon is a sliver and should be ignored
+  SLIVER_RATIO: 0.0001, // ratio of length and area to determine if the polygon is a sliver and should be ignored
+  verifyNoUpdates: process.env.VERIFY_NO_UPDATES !== undefined,
+};
+const configV2 = {
```

Comment:
Will that V2 become the default version once merged officially?

Reply:
Yes, this will be released with the featureflag ```?aggregatedViewFF=true```  to use the json files built from this config. Once we remove the feature flag this will be the default config